packages/dripsy/src/css/ssr-component.tsx

```diff
@@ -1,7 +1,7 @@
 /** @jsx jsx */
 import { jsx, SxProps } from 'theme-ui'
 import React, { ComponentProps, ComponentType, Fragment } from 'react'
-import type { ResponsiveSSRStyles } from '.'
+import { ResponsiveSSRStyles } from '.'
```

Comment:
why remove `import type`? I also just realized there may be a circular import here.

Reply:
Github is saying this is outdated, so I might be looking at an old file.